airflow/api_fastapi/common/db/dags.py

```diff
@@ -45,3 +45,13 @@
     )
     .order_by(DagModel.dag_id)
 )
+
+dagruns_select_with_state_count = (
+    select(
+        DagRun.dag_id,
+        DagRun.state,
+        func.count(DagRun.state),
+    )
+    .group_by(DagRun.dag_id, DagRun.state)
+    .order_by(DagRun.dag_id)
+)
```

Comment:
Oh okay, I was wondering if we keep all dag-related resources (dag, dag code, dag stats etc.) under same file since we've few queries. But yes, one file per entity works too. Updated PR, moved query to new file `dag_runs.py` 👍🏽